File: android/lib/src/main/java/duck/cameras/android/model/PtzStatus.java
package duck.cameras.android.model;

import java.io.Serializable;

public class PtzStatus implements Serializable {
    public Position position;
    public MoveStatus moveStatus;

    public PtzStatus() {
        position = new Position();
        moveStatus = new MoveStatus();
    }

    public static class Position implements Serializable {
        public Float x;
        public Float y;
        public Float z;
    }

    public static class MoveStatus implements Serializable {
        public String panTilt;
        public String zoom;
    }
}

File: windows/Model/PtzStatus.cs
namespace Duck.Cameras.Windows.Model
{
    public class PtzStatus
    {
        public PtzPosition Position;
        public PtzMoveStatus MoveStatus;

        public PtzStatus()
        {
            Position = new PtzPosition();
            MoveStatus = new PtzMoveStatus();
        }
    }

    public class PtzPosition
    {
        public float? X;
        public float? Y;
        public float? Z;
    }

    public class PtzMoveStatus
    {
        public string PanTilt;
        public string Zoom;
    }

}
